### PR TITLE
Add margin metric

### PR DESCRIPTION
**Pull request by bonomat**
This metric shows total margin locked up by party and counterparty for open positions only.

Looks like this

```
# HELP margin_counterparty_quantities Total position margin of our counterparties on ItchySats.
# TYPE margin_counterparty_quantities gauge
margin_counterparty_quantities 246994
# HELP margin_quantities Total position margin on ItchySats.
# TYPE margin_quantities gauge
margin_quantities 493987
```

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]
should do the job!

It's unfortunate that we have to calculate margin in all of these places, and copy-paste that code around, but there's not much we can do about it until we store the margin in the db (which you've created a ticket for already).

File: daemon/src/position_metrics.rs [hunk 1]
```diff
@@ -343,6 +389,24 @@ mod metrics {
             .unwrap()
         });
 
+    static MARGIN_QUANTITY_GAUGE: conquer_once::Lazy<prometheus::IntGauge> =
+        conquer_once::Lazy::new(|| {
+            prometheus::register_int_gauge!(
+                "margin_quantities",
```
Comment (luckysori) [inline daemon/src/position_metrics.rs]: 🙃 I don't remember the naming conventions around metrics, but what about `margin` and `margin_counterparties` (or `margin_quantity` and `margin_counterparties_quantity` if the suffix is needed).

**Review by luckysori — APPROVED** [review]
LGTM!

File: daemon/src/position_metrics.rs (hunk 1 above)
Comment (bonomat) [reply daemon/src/position_metrics.rs]: Thanks for pointint this out. The convention is that the last word should indicate the type, e.e. in this case it should be `margin_satoshis`. 

I will fix this.

Comment (luckysori) [reply daemon/src/position_metrics.rs]: Ah cool, that makes sense!

**Pull request merged**